docker-compose.base.yml

```diff
@@ -29,6 +29,11 @@ services:
                         path /capture*
                     }
 
+                    @flags {
+                        path /flags
+                        path /flags*
+                    }
```

Comment:
this puts the flags service behind the reverse proxy that's handling our other rust services.